Mathlib/LinearAlgebra/Dual.lean

```diff
@@ -563,6 +563,32 @@ theorem forall_dual_apply_eq_zero_iff (v : V) : (∀ φ : Module.Dual K V, φ v
   rfl
 #align module.forall_dual_apply_eq_zero_iff Module.forall_dual_apply_eq_zero_iff
 
+@[simp]
+theorem subsingleton_dual_iff :
```

Comment:
Should the backwards direction also be an instance?

Reply:
Thanks, I had actually meant to do this.